src/gas-hub.js

```diff
@@ -398,6 +414,44 @@ function updateBranch() {
   });
 }
 
+function handleRepoCreated(repo) {
+  return scm.getRepos()
+  .then(updateRepo)
+  .then(updateBranch)
+  .then(() => {
+    $('#new-repo-name').val('');
+    $('#new-repo-desc').val('');
+    $('#new-repo-type').val('public');
+    showAlert(`Successfully create new repository ${repo}`);
+  })
+  .catch(() => {
+    throw new Error('Repository created, but failed to show the new repository.');
+  });
+}
+
+function handleBranchCreated(branch) {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/b1257aeff34c7da1703f8a8cb7e32838448492d8/src/gas-hub.js#L432"></span>‘handleBranchCreated’ is defined but never used.</td>
    </tr>
  </tbody>
</table>